9gag/9gag.go

```diff
@@ -0,0 +1,27 @@
+package gag
+
+import (
+	"net/http"
+
+	"github.com/go-chat-bot/bot"
+)
+
+const (
+	rangomPage = "http://9gag.com/random"
+)
+
+func gag(command *bot.Cmd) (string, error) {
+	res, err := http.Get(rangomPage)
+	if err != nil {
+		return "", err
+	}
+	return res.Request.URL.String(), nil
```

Comment:
I'll try to make it not follow the redirect, we are interested just in the URL itself, not the contents. This is opening the redirected page.